simple_salesforce/api.py

```diff
@@ -135,9 +142,23 @@ def __init__(
             'X-PrettyPrint': '1'
         }
 
+        self.bulk_headers = {
+            'Content-Type': 'application/xml',
+            'X-SFDC-Session': self.session_id,
+            'charset': 'UTF-8'
+        }
+
+        self.batch_headers = {
```

Comment:
Same thing here about public availability rather than internal to `_create_batch`